.github/workflows/main.yml

```diff
@@ -47,11 +47,10 @@ jobs:
       - name: go build deps
         run: make embed-files-test
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.41.1
+          version: latest
```

Comment:
Wouldn't it be better to pin to the actual latest version vs using `latest` to avoid breakages?

Thank you for upgrading the linter!!

Reply:
We shouldn't care so much about a sliding window tag for something like a linter. A breakage in this scenario is just a failed workflow that likely should be investigated. Keeping up with incremental versions for something as trivial as linting is likely something we don't want to do. My vote would to keep this with @nojnhuh's current change with `latest`.